front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -8,11 +8,15 @@ import {
   Searchbar,
 } from "@dust-tt/sparkle";
 import type {
-  AssistantTemplateTagNameType,
   SubscriptionType,
+  TemplateTagCodeType,
+  TemplateTagsType,
   WorkspaceType,
 } from "@dust-tt/types";
-import { assistantTemplateTagNames } from "@dust-tt/types";
+import {
+  isTemplateTagsArrayValid,
```

Comment:
very nit, but IMO `isTemplateTagCodeArray` is better naming, because the function is a type guard and our (informal) naming convention for those has been `is${TYPE}` so far

Reply:
Renamed! ✅